### ISSUE TITLE
Org branding needed on ambassador page, add to assistance panel

### ISSUE THREAD

**Issue by MainStreamDanny** [gpsimpact/relational_gotv#48]
The ambassador's dashboard needs to indicate which organization they are an ambassador for.

I expect that to maintain full flexibility, the app needs to be usable if someone just signed up, and doesn't have a parent org. But if they do, it should be displayed.

I'd suggest putting it in the assistance box. "You are an ambassador for <<OrgName>>. For assistance, contact..."

I'm assuming the contact for assistance person is the org's contact person?

**Comment by jakelowen** [comment]
Good idea.. Yes, the "for assitance contact" is the appointed representative to the organization.
